src/config.js

```diff
@@ -0,0 +1,4 @@
+const API_URL = process.env.REACT_APP_SECRET_CODE || "http://localhost:2000";
```

Comment:
Fail to see how this  `config.js`  file is an improvement over current code.

Reply:
I understand that `process.env.REACT_APP_SECRET_CODE` holds production API URL, right? If this is right maybe we can find a better name.

Reply:
Hi @jvalduvieco! Didn't want to bee too invasive here, just kept iterating and ended up with this. I'm trying to avoid using `development?` and `production?` conditionals and instead aggregating all the logic in a single place, inheriting values from environment variables (or defaulting to predefined values). Something similar to what [12factor](https://12factor.net/) suggests. The idea is that this makes the environments more flexible.

As for the `REACT_APP_SECRET_CODE` it's definitely a mistake on my part :). It should be `REACT_API_URL`. The `REACT_` part is mandatory - react identifies these variables and performs a static replacement just before compiling, as env variables are not available on client-side apps.

Reply:
Oh, BTW, this was the chapter of the `12factor` guide I was referring to: https://12factor.net/config